### ISSUE TITLE
Missing text when copying lot of code in Python

### ISSUE THREAD

**Issue by Escartem** [numworks/epsilon#1599]
#### Describe the bug
When copying lot of code that goes out of the screen and then paste it, the last line will be cut.

#### Screenshots
![screenshot (1)](https://user-images.githubusercontent.com/43405304/85012319-25aa0700-b163-11ea-8091-b64e9d633313.png)
![screenshot (2)](https://user-images.githubusercontent.com/43405304/85012333-29d62480-b163-11ea-9890-a5748730e9b5.png)
![screenshot (3)](https://user-images.githubusercontent.com/43405304/85012358-322e5f80-b163-11ea-8eb4-42fcadab754c.png)

#### To Reproduce
1. Go to the python app
2. Type a lot of text to goes out of the screen
3. Copy the text
4. Paste it again

#### Expected behavior
The last line should be pasted entirely

#### Environment
 - Epsilon version : 13.2.0
 - The platforms on which the problem happens: online simulator and actual device.

**Comment by RedGl0w** [comment]
You can add cariage return at middle of the line, and you'll still have the issue, that a part of what you have coppied isn't here anymore, even if the text isn't outside the windows : the real issue is that the clipboard is pretty tiny indeed (its size is 220 bytes (like it is defined [here](https://github.com/numworks/epsilon/blob/043c564a25f11503a26c2259f8e303ee21aba281/escher/include/escher/clipboard.h#L14) and [here](https://github.com/numworks/epsilon/blob/043c564a25f11503a26c2259f8e303ee21aba281/escher/include/escher/text_field.h#L58)).

**Comment by Escartem** [comment]
Thanks for explaining, but then, is not possible to set an higher size for the clipboard?

**Comment by RedGl0w** [comment]
> Thanks for explaining, but then, is not possible to set an higher size for the clipboard?

Yes, it is possible, but it will use more ram (a consequence of static allocated buffer), so I don't think numworks will change the size of the clipboard

**Comment by Escartem** [comment]
Ok, thank you for explaining all of this.

**Comment by RedGl0w** [comment]
I think you should have let it opened, for knowing the numworks' thought on having a bigger clipboard size by the way.

**Comment by Escartem** [comment]
I reopened it then

**Comment by PolarToCartesian** [comment]
I think that numworks should show an error message when you try to copy/cut a string that is longer than the clipboard's buffer. In case of cutting text, the text should not be removed. What do you think?

**Comment by RedGl0w** [comment]
@PolarToCartesian this is related to https://github.com/numworks/epsilon/issues/1657

**Comment by PolarToCartesian** [comment]
@RedGl0w You're right!

**Comment by Escartem** [comment]
It might be a great idea !
They could also add somewhere maybe in the settings the memory usage because when we ran out of memory we just can’t write anything anymore with no warning

**Comment by RedGl0w** [comment]
> It might be a great idea !
> They could also add somewhere maybe in the settings the memory usage because when we ran out of memory we just can’t write anything anymore with no warning

Not exactly the same thing, but numworks indicated they don't want it (https://github.com/numworks/epsilon/issues/1023#issuecomment-611568102)
But if they want to add it, it is already coded (in omega : https://github.com/Omega-Numworks/Omega/commit/26b624ef0dce68307d1ef988c2c1be4b36cf81eb#diff-49f53c61ac78e4fb26080e9af23a84eb)

**Comment by Escartem** [comment]
Ok then, it’s weird they don’t want to add it since it’s not a bad feature
